src/components/calcite-pagination/calcite-pagination.tsx

```diff
@@ -159,22 +161,28 @@ export class CalcitePagination {
 
     while (currentNum <= end) {
       pages.push(currentNum);
-      currentNum ++;
+      currentNum++;
     }
 
     return pages.map(page => this.renderPage(page));
   }
 
   renderPage(num) {
     return (
-      <a class={{[CSS.page]:true, [CSS.selected]: (num === this.selectedIndex) }} onClick={() => {
-        this.selectedIndex = num;
-      }}>{num}</a>
+      <a
```

Comment:
for another pr, but these should probably just be buttons if they don't go anywhere. We should also probalby think about adding aria labels to these page links like `Go to page: ${num}` or something. I think screen readers are just going to read this "Internal link, 1" "Internal link, 2" "Internal link, 3"

Reply:
Agreed